notebooks/220-license-plate-recognition/220-license-plate-recognition.ipynb

```diff
@@ -0,0 +1,408 @@
+    "\n",
+    "![flowchart.png](https://user-images.githubusercontent.com/15709723/162659593-3f620d7a-44d2-4f49-a558-94c35a244a8e.png)\n",
+    "\n",
+    "> Note: Augmentation method on image is optional and it may lead to the wrong recognition result. Thus, we recommend you to use it under special conditions like image overexposure or too dark.\n",
+    "\n",
+    "Example image data comes from [CCPD (Chinese City Parking Dataset, ECCV)](https://github.com/detectRecog/CCPD)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import sys\n",
+    "\n",
+    "import cv2\n",
+    "import matplotlib\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "from openvino.runtime import Core"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## The Model\n",
+    "\n",
+    "### Download the Model\n",
+    "\n",
+    "We use `omz_downloader`, the tool from the `openvino-dev` package to download the selected model. It will be retrieved from the cache if the model is already downloaded."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# directory where model will be downloaded\n",
+    "base_model_dir = \"model\"\n",
+    "\n",
+    "# model name as named in Open Model Zoo\n",
+    "model_name = \"license-plate-recognition-barrier-0001\"\n",
+    "\n",
+    "# it will be retrieved from the cache if the model is already downloaded.\n",
+    "download_command = (\n",
+    "    f\"omz_downloader \"\n",
+    "    f\"--name {model_name} \"\n",
+    "    f\"--output_dir {base_model_dir} \"\n",
+    "    f\"--cache_dir {base_model_dir}\"\n",
+    ")\n",
+    "\n",
+    "# this code is provided for the first download of the model\n",
+    "! $download_command"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Load the Model\n",
+    "\n",
+    "First, we create an Inference Engine. Then we read the network architecture and model weights from the .bin and .xml files to compile for the desired device. You can choose manually CPU, GPU, MYRIAD etc.\n",
+    "\n",
+    "If you wnat OpenVINO to decide which hardware offers the best performance, you need to use `AUTO`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# \"FP16\", \"FP32\" or \"FP16-INT8\"\n",
+    "precision = \"FP16\"\n",
+    "# output path for the conversion\n",
+    "converted_model_path = f\"model/intel/{model_name}/{precision}/{model_name}.xml\"\n",
+    "\n",
```

Comment:
This should be done in the cell above. You can specify what model precision you want to download by giving additional parameter --precision (see notebook 402)